ruby/red-arrow/lib/arrow/record-batch-file-reader.rb

```diff
@@ -20,6 +20,8 @@ class RecordBatchFileReader
     include Enumerable
 
     def each
+      return to_enum(__method__) unless block_given?
```

Comment:
How about to supply the size of Enumerator?

```suggestion
      return to_enum(__method__) { n_record_batches } unless block_given?
```

Reply:
It's a good idea!
I don't know any useful use-case of this (and `Enumerator#size`) but this doesn't have a performance penalty. So it's acceptable.